deployments/development-install/helmfile.yaml

```diff
@@ -15,12 +15,15 @@ releases:
                 - ocis.kube.owncloud.test
 
       - logging:
-          level: debug
+          level: info
```

Comment:
Nit: I'd suggest to leave this at `debug` by default. For the development install.

Reply:
You sure? Usually `info` tells you everything that is needed without too much noise

Reply:
I've a bit of a different experience. It probably depends on the service you're looking at. I agree that some service are already pretty chatty at `info` level. I'd still prefer to keep `debug` for the devel install. But as mentioned this is just nitpicking and a really a minor issue. I won't give a :-1: on this if you keep this change.

Reply:
No, it's oki for me. reverted to debug